src/main/java/stringcalculator/StringAddCalculator.java

```diff
@@ -0,0 +1,23 @@
+package stringcalculator;
+
+import stringcalculator.domain.Calculator;
+import stringcalculator.domain.Text;
+import stringcalculator.utils.Splitter;
+
+public class StringAddCalculator {
+    private static final int END_NUMBER = 0;
+
+    public static int splitAndSum(String input) {
+        Text text = new Text(input);
+
+        if(text.isBlank()) {
+            return END_NUMBER;
+        }
+
+        Splitter splitter = new Splitter();
+        String[] numbers = splitter.split(input);
```

Comment:
```suggestion
        List<String> numbers = splitter.split(input);
```
배열보단 컬렉션 사용을 추천드립니다
배열은 out of index 문제가 발생하기도 쉽고, 컬렉션에서 제공하는 다양한 기능을 활용하지 못합니다